app/models/mission.rb

```diff
@@ -13,6 +15,22 @@ class Mission < ApplicationRecord
   # kaminali default mission in per page
   paginates_per 15
 
+  # setter
+  def all_tags=(names)
+    self.tags = names.split(',').map do |name|
+      Tag.where(label: name.strip).first_or_create!
+    end
+  end
+
+  # getter
+  def all_tags
+    tags.map(&:label).join(',')
+  end
+
+  def self.ransackable_attributes(*)
+    %w[id title content status status_eq started_at ended_at created_at priority user_id title_or_tags_label_cont_any]
+  end
```

Comment:
沒有特殊原因的話用 super 後再加上想要的 attributes 應該就夠了。
```suggestion
  def self.ransackable_attributes
    super << 'title_or_tags_label_cont_any'
  end
```